server.go

```diff
@@ -791,6 +792,14 @@ func (server *Server) wireFrontendBackend(serverRoute *serverRoute, handler http
 		}
 	}
 
+	// path replace
+	if len(serverRoute.replacePath) > 0 {
```

Comment:
Yeah good catch, we could convert it from a method to a function right away. Would simplify testing.